### PR TITLE
Lk optimus updates

### PR DESCRIPTION
**Pull request by ekiernan**
Updated Optimus overview to include changes to output names and the difference in the pipeline when large vs. small datasets are used.

### CODE REVIEW

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 1]
```diff
@@ -91,10 +91,10 @@ The example configuration files also contain metadata for the reference files, d
 | annotations_gtf | Cloud path to the GTF containing gene annotations used for gene tagging (must match GTF in STAR reference). | NA |
 | chemistry | Optional string describing whether data was generated with 10x v2 or v3 chemistry. Optimus validates this string. If the string does not match one of the optional strings, the pipeline will fail. You can remove the checks by setting "force_no_check = true" in the input JSON | "tenX_v2" (default) or "tenX_v3". |
 | counting_mode | String describing whether data is single-cell or single-nucleus. Single-cell mode counts reads aligned to the gene transcript, whereas single-nucleus counts whole transcript to account for nuclear pre-mRNA. | "sc_rna" or "sn_rna" |
-| output_bam_basename | Optional string used for the output BAM file basename; the default is input_id. | NA |
+| output_bam_basename | String used as a basename for output BAM file; the default is set the string used for the `input_id` parameter. | NA |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:94]:
```suggestion
| output_bam_basename | String used as a basename for output BAM file; the default is set to the string used for the `input_id` parameter. | NA |
```

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ To see specific tool parameters, select the task WDL link in the table; then vie
 
 | Task name and WDL link | Tool | Software | Description | 
 | --- | --- | --- | ------------------------------------ | 
-| [FastqProcessing.FastqProcessing (alias = SplitFastq)](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/FastqProcessing.wdl) | fastqprocess | sctools | Partitions the input FASTQ files by CB to create an array of FASTQ files that are each ~ 30 GB. | 
+| [FastqProcessing.FastqProcessing (alias = SplitFastq)](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/FastqProcessing.wdl) | fastqprocess | sctools | Partitions the input FASTQ files by CB to create an array of FASTQ files that are each ~ 30 GB. This task is skipped if the input files are less than 30 GB. |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:128]:
```suggestion
| [FastqProcessing.FastqProcessing (alias = SplitFastq)](https://github.com/broadinstitute/warp/blob/master/tasks/skylab/FastqProcessing.wdl) | fastqprocess | sctools | Partitions the input FASTQ files by CB to create an array of FASTQ files that are each ~ 30 GB. This task is skipped if the input files are smaller than 30 GB. | 
```

**Review by kayleemathews — APPROVED** [review]
A few minor suggestions but LGTM!

**Review by khajoue2 — APPROVED** [review]
Looks good.

**Pull request merged**